diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/utils/RulesConfigYamlTest.kt

```diff
@@ -7,28 +7,36 @@ import org.cqfn.diktat.common.config.rules.getRuleConfig
 import org.cqfn.diktat.ruleset.constants.Warnings
 import org.junit.jupiter.api.Assertions
 import org.junit.jupiter.api.Test
+import java.io.File
 
 /**
  * Special test that checks that developer has not forgotten to add his warning to a diktat-analysis.yml
  * This file is needed to be in tact with latest changes in Warnings.kt
  */
-class RulesConfigJsonTest {
+class RulesConfigYamlTest {
     @Test
-    fun `read rules config json`() {
-        val allRulesFromConfig = readAllRulesFromConfig()
+    fun `read rules config yml`() {
+        compareRulesAndConfig("diktat-analysis.yml")
+        compareRulesAndConfig("diktat-analysis-huawei.yml")
+        val thirdConfig = "${System.getProperty("user.dir")}${File.separator}..${File.separator}diktat-analysis.yml${File.separator}"
+        compareRulesAndConfig(thirdConfig, "diktat-analysis.yml")
```

Comment:
It is hard to differ whether it is parent's(diktat) diktat-analysis.yml or diktat-rules

Reply:
yeah, better to print absolute path to a file here